sig-api-machinery/README.md

```diff
@@ -25,6 +25,16 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=Lj1ScbXpn
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-api-machinery)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fapi-machinery)
 
+### GitHub Teams
+* **[@kubernetes/sig-api-machinery-api-reviews](https://github.com/orgs/kubernetes/teams/sig-api-machinery-api-reviews)**
```

Comment:
A non-member will get a 404 when they click these team links. They may be confused and doubt if these teams are valid. How about adding some tips around, something like "the teams are invisible to non-members, but you can mention them anyway"?

Reply:
@cblecker can we also add the description for all of these SIG teams as [SIG-CLI readme](https://github.com/kubernetes/community/blob/709d7bf026075b1ef0092947381932eac7c6e9af/sig-cli/README.md#sig-cli-teams) once did? New comers may not know what these teams do.

BTW, SIG-CLI readme removed the team description in https://github.com/kubernetes/community/commit/d8b3e0cc2b3d03088e767c9323cdef149c48251a#diff-8fd543d285b9715c87a38944c1bd2cb5. I guess it's removed by mistake. cc @jamiehannaford

Reply:
@xiangpengzhao I see the description here: https://github.com/kubernetes/community/tree/master/sig-cli#cli-sig. Which desc do you mean?

Reply:
I mean what is in the link https://github.com/kubernetes/community/blob/709d7bf026075b1ef0092947381932eac7c6e9af/sig-cli/README.md#sig-cli-teams shows:

| Name                               | Description                                      |
|------------------------------------|--------------------------------------------------|
|@kubernetes/sig-cli-bugs            | For bugs in kubectl                              |
|@kubernetes/sig-cli-feature-requests| For initial discussion of new feature requests   |
|@kubernetes/sig-cli-proposals       | For in depth discussion of new feature proposals |
|@kubernetes/sig-cli-pr-reviews      | For PR code reviews                              |
|@kubernetes/sig-test-failures       | For e2e test flakes                              |
|@kubernetes/sig-cli-misc            | For general discussion and escalation            |

Reply:
Ah I see. Yeah, it'd be great if we could somehow add those. I'm not sure there's an easy way to do it though

Reply:
Hm.. I like the table. It does make it easier for people to navigate. Let me see what I can whip up.